pkg/gitops/generate_build.go

```diff
@@ -43,6 +43,15 @@ const (
 	buildTriggerTemplateFileName  = "trigger-template.yaml"
 	buildEventListenerFileName    = "event-listener.yaml"
 	buildWebhookRouteFileName     = "build-webhook-route.yaml"
+
+	// namespace where the bundle configuration will be searched in case it is not found in the component's namespace
+	BuildBundleDefaultNamepace = "build-templates"
+	// name for a configMap that holds the URL to a build bundle
+	BuildBundleConfigMapName = "build-pipelines-defaults"
+	// data key within a configMap that holds the URL to a build bundle
+	BuildBundleConfigMapKey = "default_build_bundle"
+	// fallback bundle that will be used in case the bundle resolution fails
+	FallbackBuildBundle = "quay.io/redhat-appstudio/build-templates-bundle:8201a567956ba6d2095d615ea2c0f6ab35f9ba5f"
```

Comment:
I'm thinking this will need to stay here for now :confused: 
This value is currently hardcoded even without the overriding logic. I think we can add a TODO to have this improved later on.